man/Makefile

```diff
@@ -247,6 +246,15 @@ $(HTMLDIR)/parser_standard_options.css: parser_standard_options.css
 $(MDDIR)/mkdocs.yml: mkdocs/mkdocs.yml
 	$(INSTALL_DATA) $< $@
 
+$(MDDIR)/source/grass_logo.svg: grass_logo.svg
+	$(INSTALL_DATA) $< $@
+
+$(MDDIR)/grass_icon.png: grass_icon.png
+	$(INSTALL_DATA) $< $@
```

Comment:
This seems unused.

Reply:
This makes sure the files are copied to the dist source directory.